test/e2e/upgrade_from_latest_test.go

```diff
@@ -50,6 +50,23 @@ func TestVSphereKubernetes120BottlerocketUpgradeFromLatestMinorRelease(t *testin
 	)
 }
 
+func TestCloudstackKubernetes120UpgradeFromLatestMinorRelease(t *testing.T) {
```

Comment:
nit: to maintain consistency with the naming. If we do a regex comparison for CloudStack tests, this would slip through since it is case sensitive
 ```suggestion
func TestCloudStackKubernetes120UpgradeFromLatestMinorRelease(t *testing.T) {
```